app/app/User.php

```diff
@@ -9,7 +9,7 @@ class User extends Eloquent
     protected $fillable = [
         'email', 'password_hash', 'search_radius_km', 
 		'longitude', 'latitude', 'remember_token',
-		'home_city',
+		'home_city','home_zipcode','home_region'
```

Comment:
Add home_country_id to the list of fillable values so you can set it in the SocialAuthController.